test/system/posts_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'application_system_test_case'
+
+class PostsTest < ApplicationSystemTestCase
+  setup do
+    @user = users(:one)
+    @post = posts(:one)
+  end
+
+  test 'posts index' do
+    visit user_posts_url(@user)
+
+    assert_selector 'img', class: 'user-image'
```

Comment:
- [ ] Kindly write a dynamic test for the profile picture for each user.

Reply:
Hello @Qoosim , thank you for your review and taking the time to contact me through slack. I don't quite understand this point, the posts_index tests are checkin that the image for the user is being displayed. There are no more users to tests. Would you kindly expand on this? What exactly do you mean by "dynamic test" ? 
Again, thank you for your time.